config/locales/es/activemodel.yml

```diff
@@ -36,7 +36,7 @@ es:
         document_type: "Tipo de documento"
         document_number: "Número de documento (incluida letra)"
         year_of_birth: "Año de nacimiento"
-      local_census_records/import:
+      local_census_record/import:
```

Comment:
Actually, I think this key is OK, and it's the `config/locales/en/activemodel.yml` file which needs to be modified, according to the console on the master branch:

```
> I18n.locale = :en
=> :en
> LocalCensusRecords::Import.human_attribute_name(:file)
=> "File"
> I18n.locale = :es
=> :es
> LocalCensusRecords::Import.human_attribute_name(:file)
=> "Archivo"
```